.travis.yml

```diff
@@ -12,4 +12,8 @@ jobs:
       before_install: docker pull thesofproject/sof && docker tag thesofproject/sof sof
       script:
         - echo -e '#!/bin/bash\nmkdir build && cd build && cmake .. && make' > build.sh && chmod +x build.sh
-        - docker run -i -t -v `pwd`:/home/sof/work/sof.git --user `id -u` sof ./build.sh
\ No newline at end of file
+        - docker run -i -t -v `pwd`:/home/sof/work/sof.git --user `id -u` sof ./build.sh
```

Comment:
Use $(...) notation instead of legacy backticked `...`

https://github.com/koalaman/shellcheck/wiki/SC2006

Reply:
It wasn't target of this PR, but as it's quite small change I add it here.